modules/blocks.php

```diff
@@ -274,24 +277,28 @@ function jetpack_business_hours_render( $attributes, $content ) {
 			if ( $today === $day ) {
 				$now = strtotime( current_time( 'H:i' ) );
 				if ( $now < $opening ) {
-					$content .= '<br />';
-					$content .= esc_html( sprintf(
-						/* Translators: Amount of time until business opens. */
+					$days_hours .= '<br />';
+					$days_hours .= esc_html( sprintf(
+					/* Translators: Amount of time until business opens. */
 						_x( 'Opening in %s', 'Amount of time until business opens', 'jetpack' ),
 						human_time_diff( $now, $opening )
 					) );
 				} elseif ( $now >= $opening && $now < $closing ) {
-					$content .= '<br />';
-					$content .= esc_html( sprintf(
-						/* Translators: Amount of time until business closes. */
+					$days_hours .= '<br />';
+					$days_hours .= esc_html( sprintf(
+					/* Translators: Amount of time until business closes. */
```

Comment:
Do we need the comment if we already have the context in `_x`?

Reply:
Yes. Without the translator comment you can't commit on WordPress.com. We could, however, switch from `_x` to `__` if we leave the translator comment there.